### ISSUE TITLE
NVDA doesn't announce typed characters in Win10 powershell

### ISSUE THREAD

**Issue by Neurrone** [nvaccess/nvda#6575]
This happens even if the announce typed characters setting is on.

**Comment by josephsl** [comment]
Hi, how about for earlier versions such as Windows 8.x? Thanks.

**Comment by Neurrone** [comment]
Am not able to test that right now, anyone has an earlier version of windows?

**Comment by Adriani90** [comment]
@bdorer, @novalis7747 could you please test in an older Windows version if this is  also reproducible?
In Windows 10, NVDA announces just "space space space" for every character I type. cc: @leonardder

**Comment by bdorer** [comment]
hmm, I'm able to test this arround 2019-04-20. At the moment I have only 
access to Windows 10.

Am 18.03.2019 um 23:00 schrieb Adriani90:

**Comment by Novalis7747** [comment]
it dowsn’t work in different versions of Win10, but it works in Win7SP1. Can’t test it with Win8 or Win8.1….

**Comment by lukaszgo1** [comment]
Will test on Win 8 and 8.1 in a few days

**Comment by bdorer** [comment]
With latest NVDA-alpha NVDA reads the first char I type and then speaks 
space space and so on.
I do not have an older windows at hand.

Am 18.03.2019 um 23:00 schrieb Adriani90:

**Comment by Adriani90** [comment]
Is this somehow similar to #9102? If yes, then I would close this one in favor of #9102 because the description is more detailed there.
cc: @codeofdusk

**Comment by bdorer** [comment]
@codeofdusk if I remember correctly you worked on powershell support, right? If so, may you could look in to this?

**Comment by codeofdusk** [comment]
Yes, I have worked recently on console support.

I can't reproduce this, could you please try a few things?
* In advanced settings, what is the state of "use the new typed character support in legacy Windows consoles"? If it's on, does disabling it change behaviour?
* In advanced settings, what is the state of "use UI Automation to access Windows Console when available"? If it's off, does enabling it change behaviour?

**Comment by Adriani90** [comment]
This seems fixed now, testing with NVDA 2024.4.1. Closing as works for me. If you are still having this issue, please comment and we can reopen.
